addon/templates/components/welcome-page.hbs

```diff
@@ -0,0 +1,122 @@
+<style>
+  body {
+    font-family: "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
+    font-size: 16px;
+    line-height: 1.35em;
+    background: #FFFBF5;
+    color: #865931;
+  }
+  html {
+    box-sizing: border-box;
+  }
+  img {
+    max-width: 100%;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  .cf:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+  p {
+    margin: 0 0 .75em;
+  }
+  h2 {
+    color: #dd6a58;
+    margin-top: 1em;
+    font-size: 1.75em;
+    line-height: 1.2
+  }
+  a:link, a:visited {
+    color: #dd6a58;
+    text-decoration: none;
+  }
+  a:hover, a:active {
+    color: #c13c27;
+  }
+  .tomster {
+    width: 40%;
+    float: left;
+  }
+  .welcome {
+    width: 60%;
+    float: left;
+  }
+  .welcome-box {
+    margin: 2em auto 0;
+    max-width: 960px;
+  }
+  .welcome ol {
+    list-style: disc;
+    padding-left: 2em;
+    margin-bottom: .75em;
+  }
+  .welcome > ol > li {
+    padding-bottom: .5em;
+  }
+  .postscript {
+    clear: both;
+    text-align: center;
+    padding-top: 3em;
+    font-size: 14px;
+    color: #888;
+    font-style: italic;
+    line-height: 2;
+  }
+  .postscript code {
+    background-color: #F8E7CF;
+    border-radius: 3px;
+    font-family: Menlo, Courier, monospace;
+    font-size: 0.9em;
+    padding: 0.2em 0.5em;
+    margin: 0 0.1em;
+  }
+  @media (max-width: 700px) {
+    .welcome-box {
+      padding: 1em;
+    }
+    .welcome, .tomster {
+      width: 80%;
+      float: none;
+      margin: auto;
+    }
+    .tomster img {
+      width: 50%;
+      margin: auto;
+      display: block;
+    }
+    h2 {
+      text-align: center;
+    }
+  }
+  @media (max-width: 400px) {
+    .tomster img {
+      width: 60%;
+    }
+    .welcome, .tomster {
+      width: 100%;
+      float: none;
+      margin: auto;
+    }
+  }
+</style>
```

Comment:
Does this use of style in the body work across browsers? It's non standard I believe ...